homeassistant/components/romy/vacuum.py

```diff
@@ -0,0 +1,158 @@
+"""Support for Wi-Fi enabled ROMY vacuum cleaner robots.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/vacuum.romy/.
+"""
+
+
+from typing import Any
+
+from romy import RomyRobot
+
+from homeassistant.components.vacuum import StateVacuumEntity, VacuumEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, ICON, LOGGER
+from .coordinator import RomyVacuumCoordinator
+
+FAN_SPEED_NONE = "Default"
+FAN_SPEED_NORMAL = "Normal"
+FAN_SPEED_SILENT = "Silent"
+FAN_SPEED_INTENSIVE = "Intensive"
+FAN_SPEED_SUPER_SILENT = "Super_Silent"
+FAN_SPEED_HIGH = "High"
+FAN_SPEED_AUTO = "Auto"
+
+FAN_SPEEDS: list[str] = [
+    FAN_SPEED_NONE,
+    FAN_SPEED_NORMAL,
+    FAN_SPEED_SILENT,
+    FAN_SPEED_INTENSIVE,
+    FAN_SPEED_SUPER_SILENT,
+    FAN_SPEED_HIGH,
+    FAN_SPEED_AUTO,
+]
+
+# Commonly supported features
+SUPPORT_ROMY_ROBOT = (
+    VacuumEntityFeature.BATTERY
+    | VacuumEntityFeature.PAUSE
+    | VacuumEntityFeature.RETURN_HOME
+    | VacuumEntityFeature.SEND_COMMAND
```

Comment:
You specified that the vacuum supports send_command, so please implement also the [send command service](https://developers.home-assistant.io/docs/core/entity/vacuum#send_command-or-async_send_command).

Reply:
removed VacuumEntityFeature SEND_COMMAND